ui/src/app/kvm/views/KVMDetails/LxdProject/ProjectVMs/VMsActionBar/VMsActionBar.tsx

```diff
@@ -57,7 +57,7 @@ const VMsActionBar = ({
           appearance="vmTable"
           setSelectedAction={setSelectedAction}
         />
-        <span className="u-nudge-right">
+        <span className="u-nudge-right u-rotate-right">
```

Comment:
Hahaha I thought no one would notice! Anyway, you need to add the className to the `Button` to make it work

Reply:
done!